### PR TITLE
REDCORE-455 Table of content for webservice documentation

### PR DESCRIPTION
**Pull request by thangredweb**
Table of content for webservice documentation

### CODE REVIEW

File: component/admin/views/webservices/tmpl/default.php [hunk 1]
```diff
@@ -49,8 +50,12 @@ function setWebservice(client, webservice, version, folder, task)
 			var url = jQuery(this).attr('data-remote');
 			var format = jQuery(this).attr('data-remote-format');
 			var dataType = format == 'json' ? 'json' : 'text';
+			var ToC =
+				"<nav role='navigation' class='table-of-contents'>" +
```
Comment (Kixo) [inline component/admin/views/webservices/tmpl/default.php]: this should belong in the content and then you can just find the ul element and add li's to it...

File: libraries/redcore/layouts/webservice/documentationoperation.php [hunk 2]
```diff
@@ -22,7 +22,7 @@
 ?>
 <div class="container-fluid">
 	<div class="page-header">
-		<h3>
+		<h3 id="<?php echo (ucfirst($operationName)) ?>">
```
Comment (Kixo) [inline libraries/redcore/layouts/webservice/documentationoperation.php]: anchors should be in the <a> tag...

**Comment by javigomez** [comment]
Good job @thangredweb. We have added an update of the requirements at the issue. Please check in JIRA. Thanks

**Comment by javigomez** [comment]
Closing as agreed with @thongredweb

**Pull request closed without merging**